pyblish_lite/window.py

```diff
@@ -825,6 +822,10 @@ def on_was_published(self):
     def on_was_processed(self, result):
         models = self.data["models"]
 
+        for instance in self.controller.context:
+            if instance.id not in models["instances"].ids:
```

Comment:
Neat, well thought. :)

Just be careful about flushing this `ids` cache, perhaps during a reset? Otherwise you run the (albeit very small) risk of picking up ids from previous runs.